apps/src/codebridge/FileBrowser/hooks/useFileUploader.tsx

```diff
@@ -51,7 +51,8 @@ export const useFileUploader = (
         const bodyData = new FormData();
         bodyData.append('files[]', file);
 
-        const url = `/level_starter_assets/${name}/uuid/${uuid}.${fileType}`;
+        const encodedLevelName = encodeURIComponent(name);
```

Comment:
This is just a bit of tidy -- I noticed the level names with spaces were being included in the URL passed to the Fetch API. This technically works, but felt safer to explicitly encode.